package.json

```diff
@@ -82,11 +83,13 @@
     "git-url-parse": "^13.1.0",
     "globby": "^11.0.4",
     "https-proxy-agent": "^5.0.1",
+    "is-docker": "^2.2.1",
     "js-base64": "^3.7.2",
     "kebabcase-keys": "^1.0.0",
     "node-fetch": "^2.6.5",
     "node-ssh": "^12.0.0",
     "os-name": "^5.0.1",
+    "pascalcase-keys": "^1.0.1",
```

Comment:
Why do we need this?

Reply:
A lapsus from a previous experiment, thanks! 🤦🏼‍♂️